src/js/components/RangeInput/StyledRangeInput.js

```diff
@@ -64,7 +64,7 @@ const trackColorStyle = (props) => {
     else {
       defaultTrackColor = getRGBA(
         normalizeColor(props.theme.rangeInput.track.color, props.theme),
-        props.theme.rangeInput.track.opacity || 1,
+        props.theme.rangeInput.track.opacity || undefined,
```

Comment:
If we remove the fallback to `1` here will this be backwards compatible?

Reply:
I belive this should be fine becuase the original implementation was shortcircuting the normalizedAlpha inside getRGBA function to pass alpha channel value as 1 always and it was simply not honouring the 8-hex value with alpha value in the first place. with the updated code being changed to undefined allows the alpha channel value to be passed appropriately and regular 6-hex value works as well. Of course, this is my understanding. we can further think through it again just to ensure the updated code is still backward compatible.

Reply:
Thank you for clarifying, I see that a fallback opacity is built into `getRGBA` function that falls back to 1, so it seems reasonable to just have:

`getRGBA(normalizeColor(props.theme.rangeInput.track.color, props.theme), props.theme.rangeInput.track.opacity)`

I don't think we need the `|| undefined`